README.md

```diff
@@ -67,6 +67,55 @@ fastify.listen(3000, err => {
   console.log(`server listening on ${fastify.server.address().port}`)
 })
 ```
+## Custom transports
+
+By default, passing an object as options to the plugin will configure nodemailer's main transport (SMTP).
+
+If you need a custom transport, simply initialize the transport, and pass it to the plugin instead of an options object. For example, using the `nodemailer-sparkpost-transport`:
+
+```js
+const sparkPostTransport = require('nodemailer-sparkpost-transport');
```

Comment:
Can you remove the trailing semicolon, please? The whole Fastify ecosystem uses the [standard](https://standardjs.com/) styling which means no semicolons.